core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -959,6 +959,8 @@ private void onPlaybackEnded(MediaType mediaType, boolean stopPlaying) {
      */
     private void onPostPlayback(final Playable playable, boolean ended, boolean skipped,
                                 boolean playingNext) {
+        // Reset the temporary playback speed because it only referred to the last playable
```

Comment:
I'd suggest not using the phrase `last playable`, as it could be misleading. Maybe `because it is only applicable to the playback just completed.`?

When I read `last playable`, I thought it implies the `Playable playable` argument refers to the media to be played next. But in practice, the argument does refer to the media just played.

Reply:
Removed it